lib/src/migrators/calc_interpolation.dart

```diff
@@ -50,6 +55,11 @@ class _CalculationInterpolationVisitor extends MigrationVisitor {
               .toString()
               .replaceAll(match[0].toString(), noInterpolation);
         }
+
+        for (var match in varFuncArgs) {
+          newArg = newArg.replaceFirst('var()', match[0].toString());
```

Comment:
`match[0]` is typed as `String?`, but it's never actually null, so I think it makes more sense to write this as `match[0]!` (both here and in some of the existing code above)